target/linux/ramips/mt7620nand/config-4.9

```diff
@@ -203,10 +196,6 @@ CONFIG_SOC_MT7620=y
 # CONFIG_SOC_RT288X is not set
 # CONFIG_SOC_RT305X is not set
 # CONFIG_SOC_RT3883 is not set
-CONFIG_SPI=y
-CONFIG_SPI_MASTER=y
-# CONFIG_SPI_MT7621 is not set
-CONFIG_SPI_RT2880=y
```

Comment:
If you want to save some more space, you can disable jffs2 as well.

Reply:
ok, for now It is likely that the kernel_size restriction can be removed
I will submit a new commit later...